### DISCUSSION TITLE
Welcome to ansible-navigator Discussions!

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-navigator#570] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cloin** [comment]
Hello! Thanks for setting this up @ssbarnea! I am a technical marketer for Ansible Automation Platform at Red Hat. I work a lot on the automation creator experience on ansible.
